script/start

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# Run single command from Procfile
+#
+if [ ! "$1" ]; then
+  echo "Usage: $0 <process type>" 1>&2
+  exit 1
+fi
+
+PROCFILE=`dirname "$0"`/../Procfile
+CMD=`cat "$PROCFILE" | grep "^$1:" | cut -d: -f2-`
+if [ ! "$CMD" ]; then
+  echo "Process name $1 not found in Procfile" 1>&2
+  exit 2
+fi
+
+PS="$1"
+[ "$PORT" ] || PORT=3000
+export PS PORT
+
+exec /bin/sh -c "$CMD"
```

Comment:
why do we need a new shell for that?

Reply:
Because `$CMD` often contains shell variables (like `$PORT`).